sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -370,10 +370,12 @@ private static void handleAppStartProfilingConfig(
               try {
                 // We always delete the config file for app start profiling
                 FileUtils.deleteRecursively(appStartProfilingConfigFile);
-                if (!options.isEnableAppStartProfiling()) {
+                if (!options.isEnableAppStartProfiling() && !options.isStartProfilerOnAppStart()) {
```

Comment:
shouldn't this be an OR? 
```suggestion
                if (!options.isEnableAppStartProfiling() || !options.isStartProfilerOnAppStart()) {
```

Reply:
nope, if at least one of the options is true, then the profiler will start on next app start, and we need to create the config file for that